### PR TITLE
Pin the CMAKE version

### PR DESCRIPTION
**Pull request by AmintorDusko**
**Context:**
Pin CMake to 3.24.x in wheel-builder to avoid Python not found error in CMake 3.25, when building wheels for PennyLane-Lightning-GPU.

**Description of the Change:**
All wheels are [consistently](https://github.com/PennyLaneAI/pennylane-lightning-gpu/pull/75) pinned to the same CMAKE version.

**Benefits:**
When building Linux wheels for PennyLane-Lightning-GPU, the building process will have a consistent CMAKE version throughout the suite.

**Possible Drawbacks:**

**Related GitHub Issues:**

### CODE REVIEW

**Comment by AmintorDusko** [comment]
[sc-29794]

File: pennylane_lightning/_version.py [hunk 1]
```diff
@@ -16,4 +16,4 @@
    Version number (major.minor.patch[-label])
 """
 
-__version__ = "0.28.1-dev"
+__version__ = "0.28.1-dev1"
```
[the remark this answers is not in the document]
Comment (AmintorDusko) [reply pennylane_lightning/_version.py:19]: Yeah. Super weird. Thanks for catching that!

**Review by chaeyeunpark — APPROVED** [review]
Looks good to me!

**Comment by AmintorDusko** [comment]
Thank you, @chaeyeunpark and @mlxd!

**Pull request merged**